microsoft-edge/webview2/concepts/print.md

```diff
@@ -0,0 +1,736 @@
+---
+title: Printing in WebView2 apps
+description: How to print a web page in a WebView2 app.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 01/12/2023
+---
+# Print
+
+There are a few different ways to print a web page in WebView2 which give you varying levels of ease of use and control.
+
+## ShowPrintUI
```

Comment:
part of global headings redesign since the Print method doesn't work well when terse heading.
```suggestion
## The ShowPrintUI method
```